var/spack/repos/builtin/packages/py-logilab-common/package.py

```diff
@@ -33,7 +33,7 @@ class PyLogilabCommon(Package):
 
     version('1.2.0', 'f7b51351b7bfe052746fa04c03253c0b')
 
-    extends("python")
+    extends('python', ignore=r'bin/pytest')
```

Comment:
It means "don't symlink that particular file when you run `spack activate py-logilab-common`". It is documented [here](http://spack.readthedocs.io/en/latest/packaging_guide.html#extensions)